files/config/users.js

```diff
@@ -16,7 +16,7 @@
     {{- include "grahdb.security.extra-users.json" . | nindent 4 }}
     "{{ .Values.security.provisioner.username }}" : {
       "username" : "{{ .Values.security.provisioner.username }}",
-      "password" : "{bcrypt}{{ htpasswd .Values.security.provisioner.username .Values.security.provisioner.password | trimPrefix (printf "%s:" .Values.security.provisioner.username) }}",
+      "password" : {{ .Values.security.provisioner.passwordHash | default (printf "{bcrypt}%s" ( htpasswd .Values.security.provisioner.username .Values.security.provisioner.password | trimPrefix (printf "%s:" .Values.security.provisioner.username))) | quote }},
```

Comment:
Could you try extracting the bcrypt-ing in a helper template function?

Reply:
done